pytorch_lightning/utilities/__init__.py

```diff
@@ -57,47 +58,62 @@ def _module_available(module_path: str) -> bool:
 FLOAT64_EPSILON = numpy.finfo(numpy.float64).eps
 
 
-class AMPType(str, Enum):
+class LightningType(str, Enum):
```

Comment:
Can we use a better name? `StrEnum` or `LightningEnum`

fun fact: `StrEnum` will be part of the sdtlib in Python 3.10 (https://bugs.python.org/issue41816) (backport: https://github.com/clbarnes/backports.strenum)